lib/signature-ot1.js

```diff
@@ -59,7 +59,12 @@ module.exports = (accessCodeManager, errorResponse, hash, promise, util) => {
         }
 
         result.push("");
-        result.push(req.body);
+
+        if (req.body) {
+            result.push(req.body.toString("binary"));
```

Comment:
Absolutely!  It was not there when I wrote this code but it certainly is there now.